coordinator/tests/services/validation/util.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {getUniqueValues} from "../../../src/services/validation/util"
+
+describe("getUniqueValues returns correct values", () => {
+  test("when input is empty", () => {
```

Comment:
There's probably an argument to be had about whether an empty array contains no unique values, or the single unique value `undefined`. I don't think it matters for the purposes of our FHIR validation, as long as it treats `undefined` as being different to an actual value.